ci/conan/all/test_package/test_package.cpp

```diff
@@ -78,7 +78,7 @@ arrow::Status VectorToColumnarTable(const std::vector<struct data_row>& rows,
     // Indicate the start of a new list row. This will memorise the current
     // offset in the values builder.
     ARROW_RETURN_NOT_OK(components_builder.Append());
-    // Store the actual values. The final nullptr argument tells the underyling
+    // Store the actual values. The final nullptr argument tells the underlying
```

Comment:
Could you report this to the upstream?
https://github.com/conan-io/conan-center-index/blob/master/recipes/arrow/all/test_package/test_package.cpp